cmd/web3/main.go

```diff
@@ -45,6 +45,7 @@ const (
 )
 
 func main() {
+	log.SetFlags(log.Flags() &^ (log.Ldate | log.Ltime))
```

Comment:
Do we _never_ want datetime? It is useful e.g. when waiting for a receipt, or any other long running command. We were deliberate with our use of `fmt` vs `log` before, so `log.Fatal*` should be the only problem.

An alternative to this would be to return errors up the stack through `func(c *cli.Context) error` and let `package cli` handle it.

Reply:
We could even use an `App.ExitErrHandler` to control the behavior, e.g.:
```go
fmt.Fprintf(os.Stderr, "ERROR: %v", err)
System.exit(1)
```

Reply:
@treeder Maybe it makes sense to stay with timestamp - it would be a bit easier to debug some things (e.g long running or timeouts), or you think it's better to raise some errors with timestamp, some without?(I've already added ERROR prefix)

Reply:
I think it should be consistent over all app - either we should show timestamp everywhere or we should show without it - @treeder ?

Reply:
Are you saying every printed line? That will necessarily break things by producing non-parseable lines. We already have a standard for when to include the time or not that we have been following. The fatal calls are the only exception and we don't need to disturb the other call sites. We didn't really need a timestamp on fatal calls in most places, it was just convenient to call it and be done. If we do need a timestamp, we can log something first, then finish with 'ERROR: *' afterwards.

Reply:
Ditching fatal and returning errors will make things testable too. That's the cleanest option IMO, and we don't give anything up for it.

Reply:
Nope I was talking about errors only